constants/api.js

```diff
@@ -8,8 +8,10 @@ export default {
     logout: `${AUTH_URL}/logout`,
   },
   articles: {
-    getAll: `${API_URL}/articles`,
-    getChunk: (page, pageSize) => `${API_URL}/articles?page=${page}&pageSize=${pageSize}`,
+    getAll: (page, pageSize) => {
+      const baseUrl = `${API_URL}/articles`;
+      return page && pageSize ? `${baseUrl}?page=${page}&pageSize=${pageSize}` : baseUrl;
```

Comment:
`page && pageSize ?` - you feel that this bicycle becomes more and more confused?